draft-ietf-tls-tls13.md

```diff
@@ -2202,9 +2184,10 @@ hash
 signature
 : This field indicates the signature algorithm that may be used.
   The values indicate anonymous signatures, RSASSA-PKCS1-v1_5
-  {{RFC3447}}, DSA {{DSS}}, and ECDSA {{ECDSA}}, respectively.  The
-  "anonymous" value is meaningless in this context but used in
-  {{server-key-share}}.  It MUST NOT appear in this extension.
+  {{RFC3447}}, DSA {{DSS}}, and ECDSA {{ECDSA}}, respectively.
+  The use of DSA and anonymous are deprecated. The anonymous_RESERVED
+  and dsa_RESERVED values MUST NOT be offered or negotiated by any
+  implementation.
```

Comment:
I agree that "X or Y are deprecated" but this says "The use of X and Y are deprecated" so the subject is "use" not "X and Y".